tests/sample_tox_config.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-        py{27,35,36,37}-django{111,20,21,22}-drf{39,310,latest},
+        py{38}-django{32,40}-drf{39,310,latest},
```

Comment:
Maybe also add django42 as well, as it is present in almost all `tox.ini` files.

Reply:
We haven't added it yet since the tests are also verifying that django42 has been added once the modernisers have been run.